README.md

```diff
@@ -1,25 +1,23 @@
 Little Loader
 =============
 
-A lightweight, IE8+ JavaScript loader.
+A lightweight, IE8+ JavaScript loader that is **actually tested**...
```

Comment:
You should link to the test results if they are hosted anywhere. It would definitely drive the point home since I think a lot of folks might be suspicious of this very bold (but true) statement.